ArduCopter/Attitude.cpp

```diff
@@ -213,7 +213,7 @@ float Copter::get_pilot_desired_climb_rate(float throttle_control)
 }
 
 // get_non_takeoff_throttle - a throttle somewhere between min and mid throttle which should not lead to a takeoff
-float Copter::get_non_takeoff_throttle()
+float Copter::Mode::get_non_takeoff_throttle() const
```

Comment:
This almost looks like it should be in the motors library instead of Copter.

Reply:
*So* close.

I didn't do it for two reasons.  One is that Sub doesn't seem to have the equivalent.

The second was that I'd have to move that chunk of code into the libraries directory rather than just changing the namespace.